applications/FluidDynamicsHydraulicsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -18,14 +18,26 @@
 // Project includes
 #include "includes/define.h"
 #include "custom_python/add_custom_utilities_to_python.h"
-
+#include "custom_utilities/hydraulic_fluid_auxiliary_utilities.h"
 
 namespace Kratos::Python {
 
 void AddCustomUtilitiesToPython(pybind11::module& m)
 {
     namespace py = pybind11;
-
+    py::class_<HydraulicFluidAuxiliaryUtilities>(m, "HydraulicFluidAuxiliaryUtilities")
+        .def_static("CalculateWettedPetimeter", [](ModelPart &rModelPart, const Flags &rSkinFlag, const Variable<double> &rDistanceVariable, const bool IsHistorical)
+                    { return HydraulicFluidAuxiliaryUtilities::CalculateWettedPetimeter(rModelPart, rSkinFlag, rDistanceVariable, IsHistorical); })
```

Comment:
According to our style guide, please use only 4 spaces as indentation (same in the other `{ return ...` below).